CRM/Civicase/Hook/PostProcess/AttachEmailActivityToAllCases.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * Add bulk email as an activity to all the selected cases.
+ */
+class CRM_Civicase_Hook_PostProcess_AttachEmailActivityToAllCases {
+
+  /**
+   * Add bulk email as an activity to all the selected cases.
+   *
+   * @param string $formName
+   *   The class name of the submitted form.
+   * @param object $form
+   *   The submitted form instance.
+   */
+  public function run($formName, $form) {
```

Comment:
Can we add a test for this class?

Reply:
I think we can but to be honest in my opinion this holds very little logic as it just creates relationship between activity and cases so I am not sure that whether a test for this class makes sense

Reply:
I disagree a bit. I think it is not a waste to add a test for this. We should add a test to make sure that all the case Ids's passed in the URL are attached to the expected activity. That alone is some logic we need to make sure works as expected.